src/Symfony/Component/Workflow/Dumper/PlantUmlDumper.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Dumper;
+
+use Symfony\Component\Workflow\Definition;
+use Symfony\Component\Workflow\Marking;
+
+/**
+ * PlantUmlDumper dumps a workflow as a PlantUML file.
+ *
+ * You can convert the generated dot file with the plantuml.jar utility (http://plantuml.com/):
+ *
+ * php bin/console workflow:dump pull_request travis --format=puml | java -jar plantuml.jar -p  > workflow.png
+ *
+ * @author Sébastien Morel <morel.seb@gmail.com>
+ */
+class PlantUmlDumper implements DumperInterface
+{
+    const INITIAL = 'initial';
+
+    const MARKED = 'marked';
+
+    const SYMFONY_LOGO = 'sprite $sf_logo [81x20/16z] {
```

Comment:
Better & probably safer than string comparison would be to save them as fixtures & compare results with pregenerated fixture.

Reply:
I am not sure I understand here what you are proposing. Can you tell me more and I will be happy to change.

Reply:
I think I got it and I made changes accordingly.